config/helm/chart/default/values.yaml

```diff
@@ -204,6 +204,24 @@ csidriver:
         cpu: 20m
         memory: 30Mi
     builtIn: false
+  job:
+    securityContext:
+      allowPrivilegeEscalation: false
+      privileged: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: false
+      runAsUser: 0
+      capabilities:
+        drop:
+          - ALL
+      seLinuxOptions:
+        level: s0
+      seccompProfile:
+        type: RuntimeDefault
+    resources:
+      requests:
+        cpu: 300m
+        memory: 100Mi
```

Comment:
should be the same as the `provisioner`'s, I know the ticket has a comment that mentions a possible better config, however we should test and verify that that actually works, before making it default

the customer can configure it on their own if they wish, but otherwise, lets stay on the safe side